beacon-chain/rpc/prysm/v1alpha1/beacon/assignments_test.go

```diff
@@ -45,44 +44,6 @@ func TestServer_ListAssignments_CannotRequestFutureEpoch(t *testing.T) {
 	assert.ErrorContains(t, wanted, err)
 }
 
-func TestServer_ListAssignments_NoResults(t *testing.T) {
```

Comment:
This is no longer possible. This scenario was only possible when a beacon state had zero validators. All other scenarios are handled with API errors such as validator index out of bounds, validator public key not found, etc.